mobile/src/board/hooks.tsx

```diff
@@ -0,0 +1,30 @@
+import { useContext, useEffect, useState } from "react";
+
+import { Board } from "./entity";
+import { BoardRepository } from "./repository";
+
+export function useBoard(initialState: Board): Board {
+  const [board, setBoard] = useState<Board>(initialState);
+
+  useEffect(() => {
+    const mercureURL = `/.well-known/mercure?topic=${encodeURI(
+      window.location.href
+    )}`;
+
+    const eventSource = new EventSource(mercureURL);
+    eventSource.onmessage = async () => {
+      try {
+        const updatedBoard = await BoardRepository.getById(board.id);
+        setBoard(updatedBoard);
+      } catch (e) {
+        console.error(e);
```

Comment:
You got an error calling the API to refresh the board, but you say nothing to the user? They'll see an outdated board and do wrong moves. 

You should at least display a snackbar saying that something went wrong and the game isn't up to date.